### PR TITLE
New show_version call

### PR DESCRIPTION
**Pull request by lindycoder**
Will scrape the versions output and return a dict of anything it might find out
If the switches have units, they should be exposed as different units might have different installed version for swapping

Added all the layers and the compliance test along with the cisco implementation for reference

### CODE REVIEW

File: netman/adapters/switches/cached.py [hunk 1]
```diff
@@ -92,6 +92,10 @@ class BondCache(Cache):
     object_key = 'number'
 
 
+class VersionsCache(Cache):
```
Comment (fbouliane) [inline netman/adapters/switches/cached.py]: why not use Cache directly ?

Comment (lindycoder) [inline netman/adapters/switches/cached.py]: Addressed

**Comment by fbouliane** [comment]
:+1:

**Comment by godp1301** [comment]
:+1:

**Pull request merged**